### PR TITLE
Add support for `notSetValue`

### PR DESCRIPTION
**Pull request by gingur**

### CODE REVIEW

**Comment by yahoocla** [comment]
Thank you for submitting this pull request, however I do not see a valid CLA on file for you. Before we can merge this request please visit https://yahoocla.herokuapp.com/ and agree to the terms. Thanks! :smile:

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by roderickhsiao** [comment]
lgtm, @redonkulus ?

File: lib/createImmutableStore.js [hunk 1]
```diff
@@ -41,10 +41,10 @@ function mergeState(stateFragment, event, payload) {
     );
 }
 
-function get(key) {
+function get(key, notSetValue) {
```
Comment (redonkulus) [inline lib/createImmutableStore.js]: can we call this `defaultValue`, its a little easier to read.

**Comment by gingur** [comment]
Can someone with access ↻

**Comment by redonkulus** [comment]
👍

**Comment by redonkulus** [comment]
Thanks @gingur this is released in 0.5.6.

**Pull request merged**